crates/iota/src/name_commands.rs

```diff
@@ -1086,36 +1474,198 @@ impl std::fmt::Display for NameCommandResult {
                 );
                 write!(f, "{table}")
             }
-            Self::List(nfts) => {
-                let mut table_builder = TableBuilder::default();
+            Self::UnsetReverseLookup {
+                address,
+                digest: transaction,
+            } => {
+                writeln!(f, "Successfully unset reverse lookup for {address}")?;
+                write!(f, "Transaction digest: {transaction}")
+            }
+            Self::UnsetUserData {
+                key,
+                record,
+                digest: transaction,
+            } => {
+                writeln!(f, "Successfully unset key \"{key}\"")?;
+                format_name_record(f, record)?;
+                write!(f, "\nTransaction digest: {transaction}")
+            }
+            Self::UpdateMetadata {
+                record,
+                digest: transaction,
+            } => {
+                writeln!(f, "Successfully updated metadata")?;
+                format_name_record(f, record)?;
+                write!(f, "\nTransaction digest: {transaction}")
+            }
+        }
+    }
+}
 
-                table_builder.set_header(["id", "domain", "expiration"]);
+fn format_auction(f: &mut std::fmt::Formatter, auction: &Auction) -> std::fmt::Result {
+    let start_datetime =
+        DateTime::<Utc>::from(UNIX_EPOCH + Duration::from_millis(auction.start_timestamp_ms))
+            .format("%Y-%m-%d %H:%M:%S.%f UTC")
+            .to_string();
+    let end_datetime =
+        DateTime::<Utc>::from(UNIX_EPOCH + Duration::from_millis(auction.end_timestamp_ms))
+            .format("%Y-%m-%d %H:%M:%S.%f UTC")
+            .to_string();
+
+    let data = [
+        ("Start", start_datetime.to_string()),
+        ("End", end_datetime.to_string()),
+        (
+            "Current Bid",
+            auction.current_bid.balance.value().to_string(),
+        ),
+        ("Latest Bidder", auction.winner.to_string()),
```

Comment:
```suggestion
        ("Winner", auction.winner.to_string()),
```

Reply:
How about this, I will print winner if the auction is over, otherwise I will print Bidder. Good?

Reply:
But it can be either thing depending on whether the auction is over

Reply:
Well don't ask me if you are not waiting for my answer?

Reply:
I disagree, winner is enough, it does not assume the auction is over. I would revert your last commit.

Reply:
I like it better this way. It's more clear to the user when the auction is still going vs over

Reply:
@Thoralf-M @Alex6323 what do you guys think?